src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollListBoxAutomationPeer.cs

```diff
@@ -26,8 +28,19 @@ protected override List<AutomationPeer> GetChildrenCore()
             {
                 await infiniteScrollListBox.ItemsLock.ExecuteAsync(() =>
                 {
-                    // Don't return the LoadingStatusIndicator as an AutomationPeer, otherwise narrator will report it as an item in the list of packages, even when not visible
-                    value = base.GetChildrenCore()?.Where(lbiap => !(((ListBoxItemAutomationPeer)lbiap).Item is LoadingStatusIndicator)).ToList() ?? new List<AutomationPeer>();
+                    List<AutomationPeer> items = base.GetChildrenCore();
+                    if (items == null)
+                    {
+                        value = new List<AutomationPeer>();
+                    }
+                    else
+                    {
+                        var listCollectionView = (ListCollectionView)CollectionViewSource.GetDefaultView(items);
+                        // Don't return the LoadingStatusIndicator as an AutomationPeer, otherwise narrator will report it as an item in the list of packages, even when not visible
```

Comment:
I think this was a fix as part of 2018 era accessibility fixes. I don't think this custom automation peer actually worked though...
As Fernando ended up having to remove the Indicator from the collection.
(Scott Banning did the original change)

Check with Fernando, but I think you just revert back to the default AutomationPeer for this listbox??? (and delete this class)

Reply:
@dominoFire before this change, Narrator did not see the filtered version of Updates Available packages. It would say the count of installed packages even when I'm on the Updates tab. This is because the collection contains all Installed, then I UI-filter the updates-available. 

Can you respond to Rob's comment?

Reply:
Screenshared with Fernando, and he agreed it looked fine without the AutomationPeer.... deleted.